scripts/choco/tools/chocolateyinstall.ps1

```diff
@@ -5,17 +5,11 @@ $url64      = "https://github.com/kubeshop/tracetest/releases/download/v${env:ch
 $packageArgs = @{
   packageName   = $env:ChocolateyPackageName
   unzipLocation = $toolsDir
-  fileType      = 'exe'
-  url64bit      = $url64
-
-  softwareName  = 'tracetest*'
 
-  # %placeholder% will be replaced by the correct value in the CI pipeline
-  checksum      = '%checksum%'
-  checksumType  = 'sha256'
```

Comment:
From chocolatey bot:

- The 32 bit checksum and checksum type can be remove since there is no 32 bit url.